bootstrap/util.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Util functions cloud reusable.
+
+get_ansible_requirements() {
+  ansible-galaxy install -n -r plugin-requirements.yml
```

Comment:
are we missing "plugin-requirements.yml" from the repo? or is that going to be specified by each individual person. Maybe just call it 'plugins.yml'